scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/ipc_noLead_generator.py

```diff
@@ -41,6 +41,11 @@ def __init__(self, configuration):
                 self.ipc_defintions = yaml.safe_load(ipc_stream)
 
                 self.configuration['min_ep_to_pad_clearance'] = 0.2
+
+                #ToDo: find a settings file that can contain these.
+                self.configuration['paste_radius_ratio'] = 0.25
+                self.configuration['paste_maximum_radius'] = 0.25
+
                 if 'ipc_generic_rules' in self.ipc_defintions:
                     self.configuration['min_ep_to_pad_clearance'] = self.ipc_defintions['ipc_generic_rules'].get('min_ep_to_pad_clearance', 0.2)
             except yaml.YAMLError as exc:
```

Comment:
This line has a space before it in `ipc_gullwing_generator.py `.

Reply:
fixed